### ISSUE TITLE
video_clock tag missing

### ISSUE THREAD

**Issue by leinardi** [fbcotter/py3nvml#8]
Another tag missing is the `video_clock`:

```xml
		<clocks>
			<graphics_clock>300 MHz</graphics_clock>
			<sm_clock>300 MHz</sm_clock>
			<mem_clock>405 MHz</mem_clock>
			<video_clock>540 MHz</video_clock>
		</clocks>
		<max_clocks>
			<graphics_clock>2175 MHz</graphics_clock>
			<sm_clock>2175 MHz</sm_clock>
			<mem_clock>7000 MHz</mem_clock>
			<video_clock>1950 MHz</video_clock>
		</max_clocks>
```

**Comment by fbcotter** [comment]
Again, I'm not sure what you mean by it's missing. I can only query info available from the nvml drivers themselves. The clock function py3nvml calls is defined [here](https://docs.nvidia.com/deploy/nvml-api/group__nvmlDeviceQueries.html#group__nvmlDeviceQueries_1ge8bef48b0ae6c2bb4004875621532238) and the supported clock types are listed [here](https://docs.nvidia.com/deploy/nvml-api/group__nvmlDeviceEnumvs.html#group__nvmlDeviceEnumvs_1g805c0647be9996589fc5e3f6ff680c64).

**Comment by fbcotter** [comment]
Closed as similar to #7.
